cmd/definitions/type.go

```diff
@@ -120,7 +121,7 @@ type Pair struct {
 }
 
 func (p *Pair) Type() string {
-	return parseType(p.ptype)
+	return p.ptype
```

Comment:
Why this is changed?

Reply:
Because we won't share definitions across languages, so we can remove the extra type translation layer here.